sp_HumanEvents/sp_HumanEventsBlockViewer.sql

```diff
@@ -545,12 +547,15 @@ SELECT
     clientoption1 = bd.value('(process/@clientoption1)[1]', 'bigint'),
     clientoption2 = bd.value('(process/@clientoption1)[1]', 'bigint'),
     currentdbname = bd.value('(process/@currentdbname)[1]', 'nvarchar(128)'),
+    blocking_level = 0,
```

Comment:
`blocking_level` will indicate the number of processes between it and the lead blocker.